<!-- -->
<launch>

	<arg name="max_speed"  default="2.0"/>
	
	<node name="lane_detect_node" pkg="lane_detect" type="laneDetect_node.py" />
	
	<node name="lane_keeping_node" pkg="planning" type="lane_keeping_node">
		<param name="cmd_topic_name" value="/cmd"/>
		<param name="lane_keeping_speed" value="0.0"/>
		<param name="foresight_distance" value="5.0"/>
		
	</node>
	
	 <node type="base_control_node" pkg ="base_control" name="base_control_node" >
    	<param name ="stm32_port_name"   value="/dev/ttyUSB0"/>
    </node>


</launch>



